hack/install-manifest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2021 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+for crd in config/crd/bases/*.yaml; do
+  kubectl apply -f "${crd}"
```

Comment:
Isn't this done already by `make install` command?

Reply:
To run locally I only need the CRDs.